### ISSUE TITLE
AccessibilityLabels for markers doesnt work

### ISSUE THREAD

**Issue by skibos** [react-native-maps/react-native-maps#4989]
### Summary

Hello, I tried using appium to create automation tests for my app. Unfortunetelly I faced blocker. I can't catch correctly markers. It seems that adding props accessibilityLabel, accessible, testID dont do anything. I mean these accessibilitylabels of markers are not visible in appium. When I add title prop it became visible but title prop causes showing tooltip above marker which I want to avoid. Will it be fixed or is there any possibilities to don't show the tooltip?

### Reproducible sample code

```TSX
<Marker
          onPress={onPress(item)}
          coordinate={{ latitude: item.latitude, longitude: item.longitude }}
          tracksViewChanges={markerTracksChanges}
          key={`MapMarker-${index}-${item.latitude}-${item.longitude}`}
          accessibilityLabel={accessibilityKey},
          accessible,
          testID={accessibilityKey},
        >
        ....
          <Image .../>
          <Image .../>
        </Marker>
```

### Steps to reproduce

- have configured and displayed map with markers
- have configured and run appium

### Expected result

either not showing tooltip above marker when I add title prop or working accessibilitylabel without adding title prop

### Actual result

accessibilitylabel is visible in appium only when I add title parameter but it also causes showing tooltip above marker.

### React Native Maps Version

1.7.1

### What platforms are you seeing the problem on?

Android

### React Native Version

0.72.10

### What version of Expo are you using?

SDK 48

### Device(s)

Android emulator

### Additional information

Im using Expo 49 (in dropdown there isn't such version :( )

**Comment by mateki0** [comment]
Hi @skibos, thanks for reporting this. I don't have any app with Appium setup to be able to find a better solution for now. I was able to find this one, but it's quite hacky and shouldn't be used as default in situations like this. However you can check and decide if it's okay for you.

```jsx
<Marker title="SomeTitle">
<>
<YourPinComponent> -> Image in your case
<Callout tooltip={true}>
          <></>
</Callout>
</>
</Marker>
```

Callout inside Marker is disabling marker title and description, and we are using empty Callout so it won't be shown.

I will try to setup app with Appium and rn-maps to be able to find better solution.
